resources/js/Components/AuthWidget.vue

```diff
@@ -41,7 +39,8 @@ export default defineComponent({
         vertical-align: middle;
     }
 
-    .wikit-Link {
+    .cdx-docs-link {
+        @include cdx-mixin-link;
```

Comment:
Since the same is done for every component, why not just create the class rules in `app.scss`, this will lead to less duplication. Also what does the `docs` stand for in the class name?

Reply:
Thinking some more about this comment I realized that we can definitely just import and call the mixin for all link elements in `app.scss` and ended up removing the relevant css from the components and the `cdx-docs-link` class from the templates. 
That means inspecting the links doesn't show any association to cdx link but everything work well, so i'm not sure that's necessary anyway.